index.js

```diff
@@ -286,5 +286,24 @@ module.exports = {
 
     self.apos.define('apostrophe-cursor', require('./lib/cursor.js'));
 
+    self.on('apostrophe-pages:notFound', 'softRedirectWithPersona', async (req) => {
+      const soft = self.apos.modules['apostrophe-soft-redirects'];
+      const doc = await self.apos.docs.find(req, { historicUrls: { $in: personify(req.url) } }).sort({ updatedAt: -1 }).toObject();
+      if (!doc) {
+        return;
+      }
+      if (!doc._url) {
+        return;
+      }
+      if (soft.local(doc._url) !== req.url) {
```

Comment:
Is `self.apos.modules['apostrophe-soft-redirects']` always defined? Or do you need to check for it? 

```js
if (soft && soft.local(doc._url) !== req.url) {
```

Might be a silly question though.

Reply:
It is always defined as it is a core module activated in defaults.js.